youtube_dl/extractor/indavideo.py

```diff
@@ -60,8 +60,12 @@ def _real_extract(self, url):
 
         formats = [{
             'url': video_url,
-            'height': self._search_regex(r'\.(\d{3,4})\.mp4$', video_url, 'height', default=None),
+            'height': int_or_none(self._search_regex(r'\.(\d{3,4})\.mp4$', video_url, 'height', default=None)),
         } for video_url in video_urls]
+        
+        # Filter out redundant null height format
+        formats = [fmt for fmt in formats if fmt['height']]
```

Comment:
Looks like all this FLV stuff (which is what the API calls them but they are MP4s) is redundant, since it already provides the URLs in `video_url`. I have no idea why I did that, but getting rid of that seems to solve the problem.